contrib/motd

```diff
@@ -1,32 +1,32 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 import socket
-import executil
+from subprocess import run, PIPE
+from sys import stdin
+
 from sysversion import fmt_sysversion
 
-def fmt_welcome():
-    welcome = "Welcome to %s" % socket.gethostname().capitalize()
+encoding = stdin.encoding # most likely 'utf-8'
+
+def fmt_welcome(encoding=encoding):
+    welcome = "Welcome to {}".format(socket.gethostname().capitalize())
 
-    sysversion = fmt_sysversion()
+    sysversion = fmt_sysversion(encoding=encoding)
     if sysversion:
         welcome += ", " + sysversion
 
-    return welcome
+    return welcome  # returns string obj
```

Comment:
Perhaps even clearer than a comment here is a typed function definition i.e.

```
def fmt_welcome(encoding: str=encoding) -> str:
    ...
```

Reply:
Agreed. Although I left it as is for now. I'd actually like to include typing for all TurnKey python code and incorporate that as a component of testing (when we introduce tests).